pkg/client/selfhosted/selfhosted_test.go

```diff
@@ -55,6 +59,17 @@ func TestNew(t *testing.T) {
 		assert.Contains(t, err.Error(), "failed parsing url")
 	})
 
+	// t.Run("Error on missing host", func(t *testing.T) {
+	// 	opts := &Options{
+	// 		Host:   "",
+	// 		CAPath: "invalid/path",
+	// 	}
+	// 	client, err := New(ctx, log, opts)
+	// 	assert.Nil(t, client)
+	// 	assert.Error(t, err)
+	// 	assert.Contains(t, err.Error(), "host cannot be empty")
+	// })
+
```

Comment:
uncomment?

Reply:
This was my own implementation of checking for a missing host that you implemented 😄